torch/distributed/_sharded_tensor/ops/math_ops.py

```diff
@@ -0,0 +1,23 @@
+import torch
+
+
+def math_ops(types, args, kwargs):
```

Comment:
nit: This should be called `elementwise_ops` where you can just apply the op directly on shards.

Reply:
Thanks for the suggestion for the naming.